Cargo.toml

```diff
@@ -57,5 +57,5 @@ sp-runtime-interface = { version = "24.0.0" }
 
 # Local dependencies
 
-drink = { version = "0.11.1", path = "drink" }
-drink-test-macro = { version = "0.11.1", path = "drink/test-macro" }
+drink = { version = "0.12.0", path = "drink" }
+drink-test-macro = { version = "0.12.0", path = "drink/test-macro" }
```

Comment:
can you please use `"=0.12.0"` in both dependencies? without it, freezing `drink` doesn't freeze `drink-test-macro`